logging/terraform.tfvars.example

```diff
@@ -0,0 +1,19 @@
+tenancy_ocid = ""
```

Comment:
No need for example file here. We'll be passing the values from the base module. 
Instead these should also be in the root terraform.tfvars.example